packages/vitest/src/plugins/mock.ts

```diff
@@ -19,18 +19,18 @@ interface MockCodeblock {
 }
 
 const parseMocks = (code: string) => {
```

Comment:
why rewrite is needed? it did the same before (`hasCloasedMockLine` isn't needed bc loop continues after `line.includes(');')` and then moves index for the next block, so it will never goe there if not needed

Reply:
Let me check again 🙏🏽 

Maybe I could divide the PR in a micro-commits :)